app/services/threat_level/threat_assessment_runs_for_profile_service.rb

```diff
@@ -1,8 +1,9 @@
 module ThreatLevel
   class ThreatAssessmentRunsForProfileService
-    def self.get(profile)
-      ThreatAssessmentRun.joins(threat_assessments: :profiles).
-          where("profiles.id = #{profile.id}")
+    def self.get(profile_id)
+      ThreatAssessmentRun.includes(threat_assessments: :results).
+        joins(threat_assessments: :profiles).
+          where("profiles.id = #{profile_id}").order(created_at: :desc).limit(5)
```

Comment:
Right. Did that in commit https://github.com/ismop/dap/pull/156/commits/783f6e58d4172522dec9595aae54bb8ed0e4501c